actions/admin/forbidden-patterns/README.md

```diff
@@ -0,0 +1,54 @@
+# forbidden-patterns (composite action)
+
+Fail a workflow if any forbidden strings or regex patterns are found with `git grep` (tracked files only). Emits per-line GitHub **annotations** and a clean **Summary** table before failing.
+
+## Usage
+
+```yaml
+name: forbidden strings
+
+on:
+  pull_request:
+  push:
+    branches: [main, next, production]
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+
+      - name: Forbidden patterns
+        uses: RMI/actions/actions/admin/forbidden-patterns@main
```

Comment:
`actions/actions` always throws me

Reply:
Yeah. I'm thinking about splitting these out into separate repos, but that's a different discussion.